src/scripts/exclusion_term_expansion.py

```diff
@@ -51,19 +52,11 @@ class having `True` for `exclude_children`, but the child class has `False`.
 
 
 # Functions
-# TODO: use: https://bioregistry.readthedocs.io/
-#  en/latest/api/bioregistry.curie_from_iri.html?highlight=get_curie#bioregistry.curie_from_iri
-def uri_to_curie(uri: str) -> str:
-    """Takes an ontological URI and returns a CURIE. Works on the following patterns:
-    - http://.../PREFIX/CODE
-    - http://.../PREFIX_CODE"""
-    uri_list: List[str] = uri.split('/')
-    end_bit = uri_list[-1]
-    if '_' in end_bit:
-        curie = end_bit.replace('_', ':')
-    else:
-        curie = uri_list[-2] + ':' + end_bit
-    curie = curie.replace('<', '').replace('>', '')
+def uri_to_curie(uri: str, prefix_map: Dict[str, str]) -> str:
+    """Takes an ontological URI and returns a CURIE. Works on the following patterns
+    # todo: Use this solution until I get OAK to work for this use case
+    # todo: Might be able to speed this up by doing this op on whole column first"""
+    curie = curie_from_iri(uri.replace('<', '').replace('>', ''), prefix_map=prefix_map)
```

Comment:
I noticed this problem with angle brackets when I tried to use it the `bioregistry` feature you mentioned, so I opened up [an issue](https://github.com/biopragmatics/bioregistry/issues/479).

I know that eventually we want to use `oaklib`.

Reply:
*sigh*. Things take so much longer than expected. This `bioregistry` feature is too slow. I created an issue: https://github.com/biopragmatics/bioregistry/issues/480

I looked and the `oaklib` implementation only comes as class methods which require me to do a more elaborate setup and load up an ontology, and I don't have time to do that tonight if I want to produce some results. For now I'll borrow the `oak` code and repurpose it for both my PRs, but I'll need to change this soon.

Reply:
@joeflack4 no worries, it takes always longer to try and re-use code then to build it yourself for such methods. In the long term it will be worth, it, thank you for your patience! 

The ankle brackets issue: This was actually a mistake in _my_ code: I should have not served you IRIs with `<>`. It is good to know that these ankle brackets denotes "entity reference" is RDF. So, `http://purl.obolibrary.org/obo/MONDO_123` is an IRI, `<http://purl.obolibrary.org/obo/MONDO_123>` is a an entity reference in RDF (for example, a turtle or SPARQL file). I should removed these from the files i provided to you.

Seems the caching issue is also solved - for future iterations of the function please fold it in. :) Thanks!

Reply:
@matentzn Thanks, I understand. I did change the code to be even more robust though, if you want to leave this in:

Rather than a simple find/replace:
```py
    if uri.startswith('<'):
        uri = uri[1:]
    if uri.endswith('<'):
        uri = uri[:-1]
```

I can't think of any situation in which this would negatively effect things, unless we want to fail early and often, which I guess is maybe Pythonic.

I haven't utilized the `bioregistry` since it was updated a few days ago, but there's a `todo` in there for me to check out Charlie's new library. I don't think we should let that hold up a patch that will fix a bugfix though.

Reply:
Ok to all. Thank you